src/Explorer/Notebook/NotebookUtil.test.ts

```diff
@@ -1,12 +1,73 @@
 import { NotebookUtil } from "./NotebookUtil";
 import * as GitHubUtils from "../../Utils/GitHubUtils";
+import {
+  ImmutableNotebook,
+  MediaBundle,
+  CodeCellParams,
+  MarkdownCellParams,
+  makeCodeCell,
+  makeMarkdownCell,
+  makeNotebookRecord
+} from "@nteract/commutable";

```

Comment:
This works. FYI, nteract also has a `fixture` package that has pre-created notebooks in various shapes (small, big, object, json, etc).